src/CohortManager.js

```diff
@@ -0,0 +1,145 @@
+const Cohort = require('./Cohort.js')
+const Student = require('./Student.js')
+
+class Cohortmanager {
+  constructor() {
+    this.studentList = []
+    this.cohortList = []
+    this.studentId = 1
+  }
+
+  createCohort(name) {
+    if (name === null) {
+      throw new Error('A Cohort must have a name')
+    }
+    if (typeof name !== 'string') {
+      throw new Error('A Cohort must be a sequence of characters')
+    }
+    const searchedCohort = this.cohortList.find(
+      (cohort) => cohort.cohortName === name
+    )
+    if (!searchedCohort) {
+      const createdCohort = new Cohort(name)
+      this.cohortList.push(createdCohort)
+      return this.cohortList
+    }
+    throw new Error(`${name} already used for another cohort`)
```

Comment:
I really like this, but I'd return the Cohort itself rather than the entire list